src/api/app/controllers/webui/package_controller.rb

```diff
@@ -65,8 +65,11 @@ def show
 
     @is_current_rev = false
     if set_file_details
-      if @forced_unexpand.blank?
+      if @forced_unexpand.blank? && @service_running.blank?
         @is_current_rev = !@revision || (@revision == @current_rev)
+      elsif @service_running
```

Comment:
Minor thing. You could move this condition up and then remove the ```@service_running.blank?``` in line 68.

Also I just realize that there is a small mismatch between ``` @service_running.blank? ``` and ``` @service_running ```. The first one seems to care about being empty, the latter not.

And line 398 indicates it's a boolean. Maybe we should just treat it that way;-)